### PR TITLE
Feat html t3003

### PR DESCRIPTION
**Pull request by Abishek666**
Created child threads.

### CODE REVIEW

**Comment by saikrishnanp** [comment]
@Abishek666, Can you try adding auto opening of the application in browser when we run the command `npm run start`?

File: server.cjs [hunk 1]
```diff
@@ -10,16 +10,20 @@ app.use(express.static(path.join(__dirname, '/src')))
...
 })
 
 app.post('/get-hourly-details', function (req, res) {
-  const postData = req.body
-  const citydn = postData.city_Date_Time_Name
-  const hours = postData.hours
-  const data = timezone.nextNhoursWeather(citydn, hours, allCityData)
-  res.json(data)
+  const childProcess = fork('./childThread.cjs')
+  childProcess.send(['get-hourly-details', req.body, allCityData])
```
Comment (saikrishnanp) [inline server.cjs]: Can we send it as an object instead of an array?
It would increase the readability

File: server.cjs [hunk 2]
```diff
@@ -10,16 +10,20 @@ app.use(express.static(path.join(__dirname, '/src')))
 app.use(express.static(__dirname))
 app.use(express.json())
 app.get('/all-city-timezones', function (req, res) {
-  allCityData = timezone.allTimeZones()
-  res.json(allCityData)
+  const childProcess = fork('./childThread.cjs')
```
Comment (saikrishnanp) [inline server.cjs:13]: The forked process would remain open after the response is obtained. Can you try closing/killing the childProcess on getting back the response?

**Review by saikrishnanp — APPROVED** [review]

File: server.cjs (hunk 1 above)
Comment (Abishek666) [reply server.cjs]: sure Sai.... I will send it as an object

**Pull request closed without merging**